Documentation/user-guides/alerting.md

```diff
@@ -121,22 +121,22 @@ spec:
   serviceMonitorSelector:
     matchLabels:
       team: frontend
-  ruleFileSelector:
+  ruleSelector:
     matchLabels:
       role: alert-rules
       prometheus: example
 ```
 
 The above configuration specifies a `Prometheus` that finds all of the Alertmanagers behind the `Service` created with `alertmanager-example-service.yaml`. The `alertmanagers` `name` and `port` fields should match those of the `Service` to allow this to occur.
 
-Prometheus rule files are held in `RuleFile` custom resources. Use the label selector field `ruleFileSelector` in the Prometheus object to define the rule files that you want to be mounted into Prometheus.
+Prometheus rule files are held in `PrometheusRule` custom resources. Use the label selector field `ruleSelector` in the Prometheus object to define the rule files that you want to be mounted into Prometheus.
 
-The best practice is to label the `RuleFile`s containing rule files with `role: alert-rules` as well as the name of the Prometheus object, `prometheus: example` in this case.
+The best practice is to label the `PrometheusRule`s containing rule files with `role: alert-rules` as well as the name of the Prometheus object, `prometheus: example` in this case.
 
 [embedmd]:# (../../example/user-guides/alerting/prometheus-example-rules.yaml)
 ```yaml
 apiVersion: monitoring.coreos.com/v1
-kind: RuleFile
+kind: PrometheusRule
```

Comment:
The `[embedmd]` headers before each of these yaml blocks, indicate, that this is embedded by the file referenced. You need to change that file accordingly, or CI will fail.

Reply:
After changing the referenced file, simply run `make generate`, which will update the section in each markdown file.